DEPS

```diff
@@ -111,6 +111,8 @@ hooks = [
   {
     # Pull clang if needed or requested via GYP_DEFINES (GYP_DEFINES="clang=1").
     "name": "clang",
+    # Skip clang updates on Windows, where we don't use clang.
+    "condition": "not checkout_win",
```

Comment:
Did you double-check this does run this on Linux/Mac?  Just making sure this isn't being ignored and not run anywhere.

Reply:
I just checked, and it still runs the hook on Linux.